docs/README.md

```diff
@@ -1,24 +1,24 @@
 ---
 home: true
-actionText: 快速上手 →
+actionText: Quickstart →
 actionLink: /guide/
 features:
-- title: 易学易用
-  details: 仅有 6 个 api，对 redux 用户尤其友好，配合 umi 使用后更是降低为 0 API
-- title: elm 概念
-  details: 通过 reducers, effects 和 subscriptions 组织 model，简化 redux 和 redux-saga 引入的概念
-- title: 插件机制
-  details: 比如 dva-loading 可以自动处理 loading 状态，不用一遍遍地写 showLoading 和 hideLoading
+- title: Easy to learn and use
+  details: only 6 APIs, especially friendly to redux users. When using with umi, is reduced to 0 API
+- title: elm concept
+  details: organize models via reducers, effects and subscriptions
```

Comment:
organize models via reducers, effects and subscriptions in order to simplifies redux and redux-sage concepts.